pyecharts/echarts/label.py

```diff
@@ -20,7 +20,7 @@ def __init__(
 
         if chart_type != "graph":
             if formatter is None:
-                if type == "pie":
+                if chart_type == "pie":
```

Comment:
can we have a test case for this? 

the root cause of this is: we do not have unit test against it. if we have one now, it won't come back because unit test will fail it.

Reply:
Why not make these conditions to a `if` statement

Reply:
a single 'if', haha